docs(README) Add docs for cli options feature (#80)
coderaiser/minify: README.md

## Options

> For use in code you can provide options as an object with configurations for `html`, `css`, `js`, and `img` like so:

Reviewer: ```suggestion
For use in code you can provide options as a `minify` `argument` with configurations for `html`, `css`, `js` and `img` similar to:
```

};
```

For cli use these options can be provided in a JSON file named `.minify.json` like so: 

```json
{
    "html": {
        "removeAttributeQuotes": false
    },
    "css": {
        "compatibility": "*"
    },
    "js": {
        "ecma": 5
    },
    "img": {
        "maxSize": 4096
    }
}
```

> minify cli uses 'find-up' to locate this file, so it can be present in the working directory or any parent directory.  

Reviewer: ```suggestion
`minify` walking up parent directories to locate and read it’s configuration file `.minify.json`.  
```
